domain/src/main/java/org/reservationapplication/domain/repository/JDBCRepos/CoworkingSpaceRepositoryJDBC.java

```diff
@@ -84,26 +84,6 @@ public void create(CoworkingSpace coworkingSpace) {
         }
     }
 
-    public void updateCoworkingStatus(Long id) {
-        String sql = "UPDATE coworking_spaces SET availability_status = ? WHERE id = ?";
-
-        try (Connection connection = config.getConnection();
-             PreparedStatement statement = connection.prepareStatement(sql)) {
-
-            statement.setBoolean(1, false);
-            statement.setLong(2, id);
-
-            int rowsAffected = statement.executeUpdate();
-            if (rowsAffected == 0) {
-                throw new CoworkingSpaceNotFoundException(id, 404);
-            }
-        } catch (SQLException e) {
-            Loggers.USER_LOGGER.error("Something went wrong while updating the status of CoworkingSpace.");
-            Loggers.TECHNICAL_LOGGER.error("Database error:", e);
-        }
-    }
-
-
     @Override
     public Optional<CoworkingSpace> getById(Long id) {
         String sql = "SELECT * FROM coworking_spaces WHERE id = ?";
```

Comment:
i suppose we should read only active ones, which were not deleted

Reply:
how? I suppose we should add is_active = 1 to our queries. All read queries where you need active (not deleted) records.